hackathon/templates/hackathon/enrolment_email.txt

```diff
@@ -0,0 +1,36 @@
+<p>Hi {student},</p>
+
+<p>Thank you so much for registering for the {hackathon}!</p> 
+
+<p>
+    <strong>What does participation involve?</strong><br>
+    You'll be assigned to a team, and work together building a project based on the assigned theme in a limited number of days. Don't worry if you have limited coding experience, all levels are welcome, and we encourage alumni to participate!
+</p>
+
+<p>
+    <strong>What am I committing to?</strong><br>
+    We recommend at bare minimum you dedicate a minimum of 8-10 hours over the duration of the Hackathon. You will be expected to actively contribute to your team, not just observe.
+    Please check your calendar and confirm that you are available before registering as dropping out really lets your team down and the team will be a person short. 
+</p>
+
+<p>
+    <strong>!IMPORTANT</strong><br>
```

Comment:
Exclamation meant to be at the start?
Perhaps:

`<strong>IMPORTANT!</strong><br>`

This matches the next paragraph

Reply:
Updated as suggested